### ISSUE TITLE
Send VAT data to Directo

### ISSUE THREAD

**Issue by artur-beljajev** [internetee/registry#880]
`VATCode`, `TotalVAT`.

Before doing this, we need to double-check with Directo team, that `TotalVAT` is the amount.

https://github.com/internetee/registry/issues/844#issuecomment-391600115

**Comment by vohmar** [comment]
VAT fields confirmed by Directo

VATCode - internal code for vat percentage
TotalVAT - total vat sum of the invoice in EUR

**Comment by artur-beljajev** [comment]
`total vat sum` = VAT amount? Not total with VAT?

**Comment by vohmar** [comment]
yes, as I wrote this is total vat sum (in case of Estonia normally 20% of total invoice net sum)
